scripts/ci/40_e2e.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
 set -euxo pipefail
 
-yarn nx run "$1":e2e-ci
+APP="${1:-$APP}"
```

Comment:
Nonsensical self-reference :angry: 
```suggestion
APP="${1:-$APP}"
```

Reply:
No, this makes sense. Fails if neither `APP` is sett nor an argument provided.